widgetssdk/src/main/java/com/glia/widgets/survey/SurveyAdapter.java

```diff
@@ -56,8 +56,9 @@ interface SurveyAdapterListener {
     private final List<QuestionItem> questionItems = new ArrayList<>();
     private final SurveyAdapterListener listener;
 
-    public SurveyAdapter(SurveyAdapterListener listener) {
+    public SurveyAdapter(SurveyStyle initialStyle, SurveyAdapterListener listener) {
         this.listener = listener;
+        this.style = initialStyle;
```

Comment:
I guess we don't need `public void setStyle(SurveyStyle style)` in the `SurveyAdapter` class.

Reply:
As far as I  remember we need it. Because configured style is set via setStyle() function. I guess the initial style added by Davit to the constructor of SurveyAdapter contains the default setup. Please correct me if I'm wrong, @DavDo.

Reply:
Yep, @andrews-moc rightly observed, I just added this to avoid null checks before using it.

Reply:
Got it.

Reply:
Agree! It is bit confusing to have 2 methods for setting style(constructor and setStyle method), but in fact one of them is default empty style.

Reply:
@DavDo Could you please add a comment to the constructor explaining the default style setting to the code?